### ISSUE TITLE
Don't use gawk-specific features in example scripts

### ISSUE THREAD

**Issue by aevernon** [kaldi-asr/kaldi#1143]
As discussed in #1130, the awk expression that removes whitespace-only lines does not work with mawk (the version of awk that ships with Debian 8):

```bash
awk 'BEGIN{getline}($0 !~ /^#/) {print}' \
  $dir/lexicon0.txt | sort | awk '($0 !~ /^[[:space:]]*$/) {print}' \
   > $dir/lexicon1.txt || exit 1;
```

@danpovey suggested the following instead:

```bash
grep -v '^#' $dir/lexicon0.txt | awk 'NF>0' | sort > $dir/lexicon1.txt || exit 1;
```

The files below need to be updated:

- [x] egs/fisher_english/s5/local/fisher_prepare_dict.sh
- [x] egs/sre10/v1/local/dnn/fisher_prepare_dict.sh
- [x] egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh
- [x] egs/fisher_swbd/s5/local/fisher_prepare_dict.sh
- [x] egs/fisher_swbd/s5/local/swbd1_prepare_dict.sh
- [x] egs/swbd/s5/local/swbd1_prepare_dict.sh
- [x] egs/swbd/s5/local/swbd_p1_prepare_dict.sh
- [x] egs/swbd/s5b/local/swbd1_prepare_dict.sh
- [x] egs/swbd/s5c/local/swbd1_prepare_dict.sh
- [x] egs/aspire/s5/local/fisher_prepare_dict.sh

**Comment by aevernon** [comment]
Fixed by 84c4738 and 001e56e.

**Comment by danpovey** [comment]
Thanks.  I've asked someone to do this.

**Comment by aevernon** [comment]
It looks like there are still 12- to 24-hour delays when commenting on issues by email because your comment just showed up. I've notified GitHub.

**Comment by danpovey** [comment]
Addressed in #1147
